templates/base.html

```diff
@@ -182,5 +182,6 @@
   {% include "partial/jsonld.html" %}
 
   {% block additional_js %}{% endblock %}
+  {% include 'partial/github.html' %}  
```

Comment:
Following other plugins the if goes here so we avoid including a file that wont be used.